docs/configuration/tpu/README.md

```diff
@@ -0,0 +1,97 @@
+# **TPU Optimization Tips**
+
+This doc serves as a collection of handy tips for optimizing your vLLM on TPU workload.
+
+### **Get started**
+
+Looking for setup and installation instructions? Find them [here](https://docs.vllm.ai/en/latest/getting_started/installation/google_tpu.html).
```

Comment:
Please use relative md links for internal docs references (http links will break in future releaes)